CDS Maps/Individual Maps/CDS-to-FHIR/nNGM-CDS-to-FHIR-Sonstige Therapie.map

```diff
@@ -203,7 +202,7 @@ group CreatePatientSonstigeTherapie(source operations: BackboneElement, target b
 group TransformPatientSonstigeTherapie(source operations: BackboneElement, target tgt: Patient, target composition: Composition, target section: BackboneElement)
 {
     // Mapped profile with profile URL
-    operations -> tgt.meta as meta collate, meta.profile = 'http://uk-koeln.de/fhir/StructureDefinition/Patient/nNGM';
```

Comment:
The group for mapping the patient here can be deleted

Reply:
410